test/extended/prometheus/prometheus.go

```diff
@@ -580,8 +580,11 @@ var _ = g.Describe("[sig-instrumentation] Prometheus", func() {
 				return false, nil
 			})).NotTo(o.HaveOccurred(), fmt.Sprintf("Did not find tsdb_samples_appended_total, tsdb_head_samples_appended_total, or prometheus_tsdb_head_samples_appended_total"))
 
-			g.By("verifying the oauth-proxy reports a 403 on the root URL")
-			err := helper.ExpectURLStatusCodeExec(ns, execPod.Name, url, 403)
+			g.By("verifying the oauth-proxy reports a 401 or 403 on the root URL")
+			err := helper.ExpectURLStatusCodeExec(ns, execPod.Name, url, 401)
```

Comment:
I would rather modify `ExpectURLStatusCodeExec()` to check a list of possible status codes, e.g.

```
func ExpectURLStatusCodeExec(ns, execPodName, url string, statusCode ...int) error {
}
```

Reply:
Good idea, I will modify it in this way. :)